src/main/webapp/i18n/en/home.json

```diff
@@ -14,7 +14,8 @@
             "usernameIncorrect": "<p class='text-primary small'>Invalid username</p>",
             "tumWarning": "<p class='text-primary small'>If you are a TUM student, your username<br />should have the format <b>ab12xyz</b>.<br />Do <b>not</b> include '<b>@mytum.de</b>' or '<b>@tum.de</b>'.</p>",
             "externalUserManagementWarning": "<span class='bold'>You have entered your password too often incorrectly :-(</span><br />Please go to <a href='{{ url }}' target='_blank'>{{ name }}</a>, sign in with your account and solve the <a href='{{ url }}' target='_blank'>CAPTCHA</a>. After you have solved it, try to login again here.",
-            "loginWarning": "Seems like you are having issues signing in :-(<br />Please go to <a href='{{ url }}'>{{ name }}</a> and try to sign in there.<br />After that, try again here."
+            "loginWarning": "Seems like you are having issues signing in :-(<br />Please go to <a href='{{ url }}'>{{ name }}</a> and try to sign in there.<br />After that, try again here.",
+            "sessionExpired": "You have been logged out, because your session expired. Please login again to return to the previous page."
```

Comment:
```suggestion
            "sessionExpired": "You have been logged out because your session has expired. Please login again to return to the previous page."
```